stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -515,7 +533,10 @@ internal class ChannelControllerImpl(
                 Result(processedMessage, null)
             } else {
 
-                logger.logE("Failed to send message with id ${newMessage.id} and text ${newMessage.text}: ${result.error()}", result.error())
+                logger.logE(
+                    "Failed to send message with id ${newMessage.id} and text ${newMessage.text}: ${result.error()}",
+                    result.error()
+                )
```

Comment:
@carterhudson which editor are you using and did you run `ktlint applyToIDEAProject`?

Reply:
I'm using Android Studio with the Project code style selected and a pre-commit hook that requires ktlintFormat to be run.

Reply:
Which command do you use to format the code into your pre-commit hook?
Using `./gradlew ktlintFormat` doesn't format those lines in the current project.

Reply:
Formatting those lines is a result of the Project codestyle I'm using. It's possible mine differs if we're not tracking it in version control.